README.md

```diff
@@ -167,13 +167,114 @@ wrapping a component you intend to style, you'll need to make sure you accept
 the `className` as a prop and apply it to where you want the styles applied in
 your custom component (normally the root element).
 
-##### ...styles
```

Comment:
This section was more to describe the arguments that you can pass to `glamorousComponentFactory`.

Hense why I wanted to clean up the docs a little bit 😅